src/index.js

```diff
@@ -72,7 +72,7 @@ kalamata.expose = function(model, _opts_) {
 
     function configureEndpoints() {
 
-        app.get(options.apiRoot + opts.endpointName, function(req, res, next) {
+        app.get(options.apiRoot + opts.endpointName, async function(req, res, next) {
```

Comment:
async/await will break versions of Node.js older than v7 which is still widely used on a lot of servers.

Ideally we use vanilla promises here instead.